clint/textui/prompt.py

```diff
@@ -88,3 +88,95 @@ def query(prompt, default='', validators=None, batch=False):
             return user_input
         except Exception as e:
             puts(yellow(e.message))
+
+
+
+def options(prompt, options, default=None, batch=False):
+    '''
+
+    :param prompt:
+    :param options:
+        this can be either a list of strings, in which case it will be presented like:
+        prompt:
+            (1) this is the first string
+            (2) this is the second string
+            (3) this is the third string
+
+            please select 1-3:
```

Comment:
OK, I actually didn't add in the "please select 1-3" text, but I suppose it could be put in pretty easily.